kubernetes-typescript/index.ts

```diff
@@ -12,3 +14,14 @@ const deployment = new k8s.apps.v1.Deployment("nginx", {
     }
 });
 export const name = deployment.metadata.name;
+
+//
+// This is equivalent code using the kx package.
+//
+const pb = new kx.PodBuilder({
+    containers: [{ name: "nginx", image: "nginx" }]
+});
+const deploymentKx = new kx.Deployment("nginx-kx", {
+    spec: pb.asDeploymentSpec({ replicas: 1 }),
+});
+export const nameKx = deploymentKx.metadata.name;
```

Comment:
I was thinking we would just add the `import` but not change the template itself.  I still think that's probably right - so that the template remains minimal and simple.  Also given things are in preview and likely to change - possibly don't want to inject this code yet into all new projects.

FWIW - that's what we do for the AWS templates as well - we import `awsx` so it's there and available, but don't use it in the default code.

Reply:
Levi, if we're not putting in code and only import statements per luke's rec, then disregard my comment about the service